apps/test/unit/templates/ShowCodeToggleTest.js

```diff
@@ -213,6 +213,25 @@ describe('ShowCodeToggle', () => {
         expect(studioApp().showGeneratedCode).to.have.been.called;
       });
     });
+
+    describe("And studioApp() is subsequently initialized with enableShowCode turned off", () => {
```

Comment:
I'm not a big fan of this naming scheme -- the leading "and"s makes it unclear what scenario you're actually testing.